### ISSUE TITLE
Docker for Windows crashes.  Can't get it to NOT load at startup.

### ISSUE THREAD

**Issue by drhoads** [docker/for-win#3247]
- [ ] I have tried with the latest version of my channel (Stable or Edge)
  - [X ] I have uploaded Diagnostics
  - Diagnostics ID: 8173E999-2510-42D5-B256-0386041316F5/20190117062230

### Expected behavior
Docker not to load at bootup
### Actual behavior
Docker tries to load, and then crashes
### Information

  - Windows Version: 10 pro
  - Docker for Windows Version:

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by drhoads** [comment]
Tried this and finished it.  Rebooted.  After login I got 6 error boxes stacked on each other saying Docker Service is not running and asking if I want to Start or Quit.  If I click Quit I get  bunch more Docker not running error boxes.  I have the settings to not Start Docker when you log in.  I don’t need Docker stealing all the CPU cycles everytime I reboot or start.  Takes a long time to load.  I have to go to TaskManager and stop the processes then go to the Taskbar and Quit.  I don’t mind having Docker start on my big memory desk machine but not on my laptop.

**Comment by drhoads** [comment]
I tried to stop all docker processes in TaskManager, then followed the instruction through Windows Security and then powershell (admin) but evidently the vmcompute had never stopped, so maybe I needed to stop that service before going into Windows Security settings.  Anyway, after a reboot, Docker is still trying to load, get the message that Docker is not loaded do I want to start it.  Doesn't matter what I choose I get a Docker Crash.
